lib/livebook_web/live/hub/edit/team_component.ex

```diff
@@ -82,6 +85,23 @@ defmodule LivebookWeb.Hub.Edit.TeamComponent do
                 >
                   <.remix_icon icon="key-2-fill" /> Display Teams key
                 </a>
+                <%= if @is_default do %>
+                  <a
+                    phx-click={JS.push("remove_as_default")}
+                    phx-target={@myself}
+                    class="hover:text-blue-600 cursor-pointer"
+                  >
+                    <.remix_icon icon="star-fill" /> Remove as default
+                  </a>
+                <% else %>
+                  <a
+                    phx-click={JS.push("mark_as_default")}
+                    phx-target={@myself}
+                    class="hover:text-blue-600 cursor-pointer"
+                  >
+                    <.remix_icon icon="star-line" /> Mark as default
+                  </a>
+                <% end %>
```

Comment:
Instead of having two actions on the specific hub, we could have "Use as default" on all hubs, including the personal one. When setting the personal one we would just remove the id from storage, since it's the default by default :)

Reply:
I think I prefer this version because otherwise we also need to show this button on the "personal hub" only if you have a team or not.  But I am not sure...

In other words, let's give this a try and we will see how it changes later.

Reply:
> otherwise we also need to show this button on the "personal hub" only if you have a team or not

I don't think we have to, I'm thinking either say it's the default or have a way to make it the default. Basically the same we used to have for file systems, before we removed that from settings. That said, whichever you prefer :)

Reply:
Yeah, let's keep it as is then. Especially because there is no "action area" where we have these buttons in the personal hub.